tests/common.go

```diff
@@ -5636,6 +5667,79 @@ func DeleteBucket(provider string, bucketName string) {
 	})
 }
 
+// DeleteSnapshotsForVolumes deletes snapshots for the specified volumes based on cluster provider
+func DeleteSnapshotsForVolumes(volumes []string) error {
```

Comment:
Shouldn't this be part of volume driver?

Reply:
Done. moved to volume driver